packages/plugin-astro/src/index.ts

```diff
@@ -113,7 +116,7 @@ export function createPlugin(options: Record<string, OramaOptions>): AstroIntegr
         config = cfg
       },
       'astro:build:done': async function ({ pages, routes }: AstroBuildDoneArgs): Promise<void> {
-        const assetsDir = joinPath(config.outDir.pathname, 'assets')
+        const assetsDir = joinPath(config.outDir.pathname, 'assets').slice(isWindows ? 1 : 0);
```

Comment:
This is needed because the pathname received by astro on windows is malformed.
e.g.:

```
'/C:/Users/Matteo_Pietro_Dazzi/Downloads/molecular-magnitude/dist/'
```